lib/reporters/base.js

```diff
@@ -283,68 +282,31 @@ exports.list = function(failures) {
  */
 
 function Base(runner) {
-  var stats = (this.stats = {
-    suites: 0,
-    tests: 0,
-    passes: 0,
-    pending: 0,
-    failures: 0
-  });
   var failures = (this.failures = []);
 
   if (!runner) {
-    return;
+    throw new Error('Missing runner argument');
   }
+  this.stats = getStats(); // attached so Reporters have a reference
```

Comment:
Why isn't this
`this.stats = runner.stats;`

Reply:
Yep good call, have made change + build passing. 
If theres nothing else could you approve it please and I will "Rebase and merge".